receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -122,13 +120,50 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 				ghs.logger.Sugar().Errorf("error getting contributor count for repo %s", zap.Error(err), repo.Name)
 			}
 			ghs.mb.RecordGitRepositoryContributorCountDataPoint(now, int64(contribs), name)
+
+			// Get Pull Request data
+			prs, err := ghs.getPullRequests(ctx, genClient, name)
+			if err != nil {
+				ghs.logger.Sugar().Errorf("error getting pull requests for repo %s", zap.Error(err), repo.Name)
+			}
+
+			var merged int
+			var open int
+
+			for _, pr := range prs {
+				if pr.Merged {
+					merged++
+
+					age := getAge(pr.CreatedAt, pr.MergedAt)
+
+					ghs.mb.RecordGitRepositoryPullRequestMergedTimeDataPoint(now, age, name, pr.HeadRefName)
+
+				} else {
+					open++
```

Comment:
What about PRs that were not merged but were instead closed without merging? Are they counted as open here?

Reply:
Oh I think I see, in `helpers.go` we're only retrieving Open and Merged PRs, so we disregard Closed PRs.

Reply:
I see this is marked as resolved, but just in case, that's correct. We only deal with Open and Merged PRs.